freqtrade/vendor/qtpylib/indicators.py

```diff
@@ -557,8 +557,71 @@ def stoch(df, window=14, d=3, k=3, fast=False):
 
     return pd.DataFrame(index=df.index, data=data)
 
+
 # ---------------------------------------------
 
+def super_trend(df, period=14, multiplier=3):
+    """
+    compute the super trend indicator
+    adds following to dataframe:
+    'st': calculated super trend
+    'stx': super trend direction: up or down
+    """
+    ohlc = ['open', 'high', 'low', 'close']
+
+    df['atr'] = atr(df)
+
+    # Compute basic upper and lower bands
+    df['basic_ub'] = (df[ohlc[1]] + df[ohlc[2]]) / 2 + multiplier * df["atr"]
+    df['basic_lb'] = (df[ohlc[1]] + df[ohlc[2]]) / 2 - multiplier * df["atr"]
+
+    # Compute final upper and lower bands
+    df['final_ub'] = 0.00
+    df['final_lb'] = 0.00
+    for i in range(period, len(df)):
+        df['final_ub'].iat[i] = df['basic_ub'].iat[i] \
```

Comment:
Please don't use `if expressions` with such complicated conditions, the traditional `if` statement is more readable